data/sensor_units.py

```diff
@@ -0,0 +1,50 @@
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Library General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# sensor_types.py
+# Copyright (C) 2011 Simon Newton
+# The sensor type enums. These aren't stored in the datastore because I'm lazy
+# and for now we don't have to search on them
+
+SENSOR_UNITS = {
+    0x00 : "None",
+    0x01 : "Centigrade",
+    0x02 : "Volts_dc",
```

Comment:
We probably want these pretty-printed given we'll be displaying them, so no underscores and correct capitalisation of abbreviations/acronyms.